server/testing/src/main/java/org/apache/james/util/docker/JamesContainer.java

```diff
@@ -161,8 +134,18 @@ public boolean tryConnect(int port) {
     }
 
     @Override
-    public Statement apply(Statement statement, Description description) {
-        return container.apply(statement, description);
+    public Statement apply(Statement base, Description description) {
```

Comment:
why this change ?

Reply:
container::apply is deprecated.

Cf: static policy of TestContainers guys.